geopandas/geodataframe.py

```diff
@@ -1516,18 +1516,32 @@ def merge_geometries(block):
         return aggregated
 
     # overrides the pandas native explode method to break up features geometrically
-    def explode(self, column=None, **kwargs):
+    def explode(self, column=None, ignore_index=False, add_multiindex=True, **kwargs):
```

Comment:
```suggestion
    def explode(self, column=None, ignore_index=False, add_multiindex=None, **kwargs):
```

I would temporarily use `None` here but defaulting to True. In that case, we raise the `FutureWarning`. But the user can specify `add_multiindex=True` which silents the warning.